src/actions/index.js

```diff
@@ -118,3 +123,29 @@ export function fetchProjects(parms) {
       })
   }
 }
+
+
+export function fetchPublications() {
+  return dispatch => {
+    map((key) => {
+      addIndex(forEach)(
+        (project, idx) => {
+          dispatch(setState(`publications.${key}.projects.${idx}.publications`, project.publications))
+          dispatch(setState(`publications.${key}.projects.${idx}.slug`, project.slug))
+
+          if (project.slug) {
```

Comment:
Is this supposed to fetch a link and avatar for each project under publications? Doesn't look like the projects are linked.